Framework/Adxstudio.Xrm/Diagnostics/Metrics/IfxMetricsReporter.cs

```diff
@@ -112,9 +112,6 @@ private static class MdmDimensionNames
 #if CLOUDINSTRUMENTATION
 		readonly ConcurrentDictionary<IAdxMetric, MeasureMetric10D> _metricsCache = new ConcurrentDictionary<IAdxMetric, MeasureMetric10D>();
 #endif
-		readonly string _mdmAccountName, _mdmNamespace, _geo, _tenant, _role, _roleInstance, _org, _portalType, _portalId, _portalApp, _portalUrl, _portalVersion;
```

Comment:
I've noticed these fields (line 115 and 116) are indeed being used inside the `#if CLOUDINSTRUMENTATION` directive of the `IfxMetricsReporter()` method. Moving the `#endif` from before line 115 to after line 116 should do the trick.